services/ui-src/src/utils/other/export.test.tsx

```diff
@@ -53,6 +56,56 @@ describe("Test rendering methods", () => {
     );
     expect(result.props.children.id).toEqual("email-field-id");
   });
+
+  test("renderDrawerDataCell renders ilos responses", () => {
+    const mockFormField: FormField = {
+      id: "plan_ilosUtilizationByPlan",
+      props: {
+        choices: [],
+      },
+      type: "checkbox",
+      validation: "checkbox",
+    };
+
+    const mockPlan: AnyObject = [
+      {
+        id: "mock-id",
+        plan_ilosUtilizationByPlan: [
+          {
+            key: "123",
+            value: "mock-ilos",
+          },
+        ],
+        plan_ilosUtilizationByPlan_123: "N/A",
+      },
+    ];
+
+    const cells = renderDrawerDataCell(mockFormField, mockPlan, "drawer");
+    const ul = cells[0].props.children;
+    const li = Array.from(ul.props.children)
+      .flat()
+      .filter((el) => (el as ReactJSXElement)?.type === "li");
+    expect(cells.length).toBe(1);
+    expect(li.length).toBe(3);
+  });
+
```

Comment:
Adding the opposite scenario would be good too:

```suggestion

    test("renderDrawerDataCell renders without ilos responses", () => {
    const mockFormField: FormField = {
      id: "mock",
      props: {
        choices: [],
      },
      type: "checkbox",
      validation: "checkbox",
    };

    const mockPlan: AnyObject = [
      {
        id: "mock-id",
        mock: [
          {
            key: "123",
            value: "mock-ilos",
          },
        ],
        mock_123: "N/A",
      },
    ];

    const cells = renderDrawerDataCell(mockFormField, mockPlan, "drawer");
    const Component = () => cells[0];
    const { container } = render(<Component />);
    expect(container.querySelectorAll("li").length).toBe(2);
    expect(screen.queryByText("mock-ilos: N/A")).not.toBeInTheDocument();

```